app.ts

```diff
@@ -0,0 +1,414 @@
+// Get the container from the html where the data will be stored
+let recordNav: any = document.querySelector("#record-navigation-container"); //Navigation area;
+let headingColumns: any = document.querySelector("#column-headings-container"); //Headings;
+let infoColumns: any = document.querySelector("#info-columns-container"); //Information;
+let count = 1;
+let debounce = (func: any, delay: any) => {
+  let timer: any;
+  return function () {
+    clearTimeout(timer);
+    count++;
+
+    timer = setTimeout(() => {
+      func();
+      count = 1;
+    }, delay);
+  };
+};
+
+// Creates the navigation.
+function createNavigation() {
+  recordNav.innerHTML = `
+  <div class="navigation-btns">
+    <button class="first-page-btn">First Page</button>  
+    <button class="previous-records-btn">Previous</button>
+    <button class="next-records-btn">Next</button>
+    <button class="last-page-btn">Last Page</button>  
+    <button onclick="recordSelection()" id="confirmation-btn">Get Record</button>
+  </div>
+  <div class="current-page-container">
+    <p class=current-page></p>
+  </div>
+
+  `;
+}
+
+// The functionality to move through an amount of records.
+function nextPrevious(firstId: any) {
+  let navigation = document.querySelector(".navigation-btns");
+
+  let firstPage: any = document.querySelector(".first-page-btn");
+  let lastPage: any = document.querySelector(".last-page-btn");
+  let nextBtn: any = document.querySelector(".next-records-btn");
+  let previousBtn: any = document.querySelector(".previous-records-btn");
+
+  let currentPage: any = document.querySelector(".current-page");
+  let finalId: any;
+
+  if (recordNav.contains(navigation) === true) {
+    let resizeScreen = () => {
+      let amountRecord: any;
+
+      if (finalId >= 999999) {
+        if (window.innerHeight > 500) {
+          finalId = 999999;
+          firstId = finalId - 14;
+        } else if (window.innerHeight <= 500 && window.innerHeight > 350) {
+          finalId = 999999;
+          firstId = finalId - 10;
+        } else if (window.innerHeight <= 350 && window.innerHeight > 170) {
+          finalId = 999999;
+          firstId = finalId - 2;
+        } else if (window.innerHeight <= 170) {
+          finalId = 999999;
```

Comment:
`finalId` is being repeated multiple times in the if statements. This is a good sign to either create a method or just simply move that line outside of the if-else statements. Code refactoring basically :P